internal/zinc-core/src/main/scala/sbt/internal/inc/Compile.scala

```diff
@@ -131,6 +134,8 @@ private final class AnalysisCallback(
     options: IncOptions
 ) extends xsbti.AnalysisCallback {
 
+  private type ConcurrentSet[A] = ConcurrentHashMap.KeySetView[A, java.lang.Boolean]
```

Comment:
In case you wonder, there is no `ConcurrentSet` type in Java (nor Scala), and this how one is supposed to create one.